MovieSharing/ContentView.swift

```diff
@@ -8,13 +8,19 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    // MARK: - Properties
+    
     var body: some View {
         Text("Hello there!!")
             .padding()
     }
 }
 
 struct ContentView_Previews: PreviewProvider {
+    
+    // MARK: - Properties
```

Comment:
this is good :) I think, at the moment we don't put MARKS in the preview, but it of course is not wrong. Maybe instead (or additionally) you could add another "MARK: - Previews" above this preview.

The preview (so you know) is what makes you see how your ContentView() looks and behaves like directly in the IDE without having to start the whole app and simulator. If you don't currently see the preview you can enable it in the menu in the top bar -> Editor->Canvas.